.github/scripts/main/test.sh

```diff
@@ -26,3 +26,9 @@ eval $(opam env)
 opam install lwt
 opam list
 opam config report
+
+# Make sure opam init (including the the sandbox script) works in the absence of OPAMROOT
+unset OPAMROOT
+rm -r ~/.opam
+opam init --bare
```

Comment:
Do we want to use the default repository (download archive for each job, then extracting it, etc.)? But i don't know if it is a good idea to work with a fake local repo, it will change default behaviour.